apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/page.tsx

```diff
@@ -1,10 +1,13 @@
 import { PurchaseDetails, TermsAndPrivacy } from '@fxa/payments/ui/server';
-import { getCartData, getContentfulContent } from '../../../../_lib/apiClient';
+import {
+  getCartData,
+  getContentfulContent,
+} from '../../../../../../_lib/apiClient';
 import { headers } from 'next/headers';
 import Image from 'next/image';
 import checkLogo from '../../../../../images/check.svg';
 import circledConfirm from '../../../../../images/circled-confirm.svg';
-import { formatPlanPricing } from '../../../../../../../../libs/payments/ui/src/lib/utils/helpers';
+import { formatPlanPricing } from '../../../../../../../../../../libs/payments/ui/src/lib/utils/helpers';
```

Comment:
Could this be updated to use the TS alias instead?

Reply:
I hope so, that is just awful!